src/app/builder/builder.component.html

```diff
@@ -25,6 +25,16 @@
       width="600"
       [src]="previewIframeSrc"></iframe>
   </div>
+  <div *ngIf="playPlaylist" id="preview-playlist-wrap">
```

Comment:
Playlist is selected along with play specific episode
<img width="1348" alt="screen shot 2017-05-26 at 11 07 40 am" src="https://cloud.githubusercontent.com/assets/2250413/26502783/ca7aa3ac-4203-11e7-8597-1eea5e834076.png">